src/rpc/rawtransaction_util.cpp

```diff
@@ -207,6 +207,9 @@ void ParsePrevouts(const UniValue& prevTxsUnival, FillableSigningProvider* keyst
                         throw JSONRPCError(RPC_INVALID_PARAMETER, "redeemScript does not match scriptPubKey");
                     }
                 }
+                if (rs.isNull()) {
+                    throw JSONRPCError(RPC_INVALID_PARAMETER, "Missing redeemScript");
```

Comment:
16250: nit: partial, need 13072 for changes in `test/functional/rpc_createmultisig.py`

Reply:
I am preparing new PR with cummulative changes for `test/functional/rpc_createmultisig.py`, will be done in next week probably, including 16250, 16026, 14180, 13541 and others

Reply:
There is missing changes: https://github.com/dashpay/dash/pull/5278